index.html

```diff
@@ -5,22 +5,101 @@
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one, two, three">
 
-		<title>Blank Template</title>
+		<title>Homework Assignment 3</title>
 
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
-
+		<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,800" rel="stylesheet">
 	</head>
 	<body>
-		<div id="container">
 			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+				<h1>Relaxr</h1>  
+				<nav class="clearfix"> 
+					<ul class="alignright clearfix"> 
```

Comment:
instead of putting the `alignright` class on the `ul`, put it on the `nav`. That way you can treat the `h1` and `nav` as your "columns". Therefore, you should then put a ` clearfix` on the `header`. 

Additionally, rather than set the `h1` to be `display: inline`, just do a `float: left` and set a width to something like `40%` or something reasonable.